### PR TITLE
Phase2 bughunt v8

### PR DESCRIPTION
**Pull request by Ronak-Searce**
- [ ] Align the no issues image and icon to center.
- [ ] Tables are not clickable if you drop all the tables and save the session and then open the saved session.
- [ ] Getting the note for index name not adhering to the regex but it does. Either its a false error or wrong message.

### CODE REVIEW

File: ui/src/app/components/summary/summary.component.scss [hunk 1]
```diff
@@ -24,7 +24,7 @@
   }
   .no-issue-container {
     width: 100%;
-    min-height: 400px;
+    min-height: 300px;
```
Comment (shreyakhajanchi) [inline ui/src/app/components/summary/summary.component.scss:27]: Its not vertically aligned: https://drive.google.com/file/d/1VJ6FPpxoS5QRgjU4R0EWDJ4WJGrwtuIi/view?usp=share_link&resourcekey=0-bdDHfglZTtk-tp83j-yX6g

**Review by shreyakhajanchi — COMMENTED** [review]
Left a comment regarding the alignment , rest all looks fine

File: ui/src/app/components/summary/summary.component.scss (hunk 1 above)
Comment (Ronak-Searce) [reply ui/src/app/components/summary/summary.component.scss]: Updated

**Review by shreyakhajanchi — APPROVED** [review]
LGTM

**Pull request merged**